README.md

```diff
@@ -19,3 +19,46 @@ quantitatively reason with them.
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
+
+-----
+
+A "good pivot" is a pivot that is placed in the middle range of an array.  According to the lecture slide on the intuition of the average case of quicksort, the probability that the randomly selected element is an element in the middle portion of the array is 1/2.
+
+When considering the median-to-three method for finding a pivot, I wrote out all permutations of three elements (e1, e2, and e3, where e1 is the first element, e2 is the middle element, and e3 is the last element).  I created a table, which looks like:
+
+| e1 | e2 | e3 |
+| -- | -- | -- |
+| 0  | 0  | 0  |
+| 0  | 0  | 1  |
+| 0  | 1  | 0  |
+| 1  | 0  | 0  |
+| 1  | 1  | 0  |
+| 0  | 1  | 1  |
+| 1  | 0  | 1  |
+| 1  | 1  | 1  |
+
+1 represents when an element is the median element, and 0 represents when an element is not the median element.
+
+Here are the different combinations e1, e2, and e3:
+
+e1 < e2 < e3,
+
+e1 < e3 < e2,
+
+e2 < e1 < e3,
+
+e2 < e3 < e1,
+
+e3 < e1 < e2,
+
+e3 < e2 < e1.
+
+Looking at the 6 different combinations of the elements, each combination has a 16.67% chance of occuring.  Each element appears as the median 2 out of the 6 combinations, which is 33.33%.  The chance of the median element occuring in the middle portion of the element is 33.33%, as there are three portions to the array (left, middle, right), and an element will be placed in the middle portion 1/3 = 33.33% of the time.  Combining these probabilities, I get 83.33%, which argues that the median-to-three method is 33.33% more likely to pick a good pivot compared to the average input method.
```

Comment:
But these combinations have different probabilities, as the middle half is twice as large as the first and last quarters.